client/cert_manager.py

```diff
@@ -0,0 +1,156 @@
+"""Create ssl certificates for all1input."""
```

Comment:
:-) Maybe remove this line, or change it to a link to all1input's version this was based on?